app/src/androidTest/java/org/mozilla/focus/activity/robots/TabsTrayRobot.kt

```diff
@@ -16,17 +16,31 @@ import org.hamcrest.Matchers.allOf
 import org.mozilla.focus.R
 
 class TabsTrayRobot {
-    fun verifyTabsOrderAndCorespondingCloseButton(vararg tabTitle: String) {
+    fun verifyTabsOrder(vararg tabTitle: String) {
         for (tab in tabTitle.indices) {
             onView(withId(R.id.sessions)).check(
                 matches(
                     hasDescendant(
                         allOf(
                             hasDescendant(
-                                allOf(
-                                    withText(tabTitle[tab]),
-                                    hasSibling(withId(R.id.close_button))
-                                )
+                                withText(tabTitle[tab])
+                            ),
+                            withParentIndex(tab)
+                        )
+                    )
+                )
+            )
+        }
+    }
+
+    fun verifyCloseTabsButton(numberOfOpenedTabs: Int) {
```

Comment:
nit: verifyCloseTabButton

Reply:
Also, I would create a simple matcher for the close tab button, in relation to a given tab (given by a parameter) which would be reusable. And for the testVisitingMultipleSites test, I would separate the close tab button (and other tabs tray elements) validation in a new test (to match testRail).

Reply:
Done!